commands/FBClassDump.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/python
+import string
+import lldb
+import fblldbbase as fb
+import fblldbobjcruntimehelpers as runtimeHelpers
+
+def lldbcommands():
+  return [
+    FBPrintMethods()
+  ]
+
+class FBPrintMethods(fb.FBCommand):
+  def name(self):
+    return 'pmethods'
+
+  def description(self):
+    return 'Print the class instance methods.'
+
+  def options(self):
+    return [
+      fb.FBCommandArgument(short='-a', long='--address', arg='showaddr', help='Print the implementation address of the method', default=False, boolean=True),
+      fb.FBCommandArgument(short='-i', long='--instance', arg='insmethod', help='Print the instance methods', default=False, boolean=True),
+      fb.FBCommandArgument(short='-c', long='--class', arg='clsmethod', help='Print the class methods', default=False, boolean=True)
+    ]
+
+  def args(self):
+    return [ fb.FBCommandArgument(arg='class or instance', type='id or Class', help='an Objective-C Class.') ]
+
+  def run(self, arguments, options):
+    cls = arguments[0]
+    if not isClassObject(cls):
+        cls = runtimeHelpers.object_getClass(cls)
+        if not isClassObject(cls):
+            raise Exception('Invalid argument. Please specify an instance or a Class.')
+
+    if options.clsmethod:
+      print 'Class Methods:'
+      printClassMethods(cls, options.showaddr)
+
+    if options.insmethod:
+      print '\nInstance Methods:'
+      printInstanceMethods(cls, options.showaddr)
+
+    if not options.clsmethod and not options.insmethod:
+      print 'Class Methods:'
+      printClassMethods(cls, options.showaddr)
+      print '\nInstance Methods:'
+      printInstanceMethods(cls, options.showaddr)
+
+def isClassObject(arg):
+    return runtimeHelpers.class_isMetaClass(runtimeHelpers.object_getClass(arg))
```

Comment:
There's simple `object_isClass` method in objc_runtime
No need to tweak around with metaclasses